components/alert_banners/README.md

```diff
@@ -6,57 +6,216 @@ author: orinevares
 ---
 
 ![Status](https://img.shields.io/badge/Recommended-Draft-orange.svg)
-> Last Updated: April 3, 2019
+> Last Updated: April 15, 2019
 
 # Alert Banners
-
-Alert banners notify users of important information or changes on a page. Typically, they appear after at the top of a page.
+Alert banners notify users of important information or changes on a page. Typically, they appear at the top of a page.
 
 ## Example
-
-<component-preview path="components/alert_banners/sample.html" height="100px" width="800px"> </component-preview>
+<component-preview path="components/alert_banners/sample.html" height="688px" width="800px"> </component-preview>
 
 ## Use This For
 **Validation**
 * Alerting users they just did something successful or if it needs to be corrected
 
 **Notification**
-* To keep users informed on important and typically time sensitive information. Users may or may not have to respond. This includes errors, warnings, or general updates.
+* To inform users on important and typically time sensitive information. Users may or may not have to respond. This includes errors, warnings, or general updates.
 
 ## Don't Use This For
-*	In line error validation for form inputs
+*	In-line error validation on form inputs
 * Popular links on your website. These should be readily available on your homepage.
```

Comment:
Not sure what this is supposed to say or why it's here...did you copy this from a different resource?